app/serializers/forest_liana/serializer_factory.rb

```diff
@@ -41,6 +41,8 @@ def self.get_serializer_name(active_record_class)
         "ForestLiana::ActionSerializer"
       elsif active_record_class == ForestLiana::Model::Segment
         "ForestLiana::SegmentSerializer"
+      elsif active_record_class == ::ForestLiana::MixpanelLastEvents
```

Comment:
I get an error on any collections list here.
Don't know why. Did I do something wrong?

I added the Mixpanel gem on my project (Live Demo using Rails 5.2),
bundle installed it:
```
mixpanel_client 4.1.6
```
Added the config:

```ruby
ForestLiana.env_secret = Rails.application.secrets.forest_env_secret
ForestLiana.auth_secret = Rails.application.secrets.forest_auth_secret

ForestLiana.integrations = {
  stripe: {
    api_key: ENV['STRIPE_SECRET_KEY'],
    mapping: 'Customer.stripe_id'
  },
  mixpanel: {
    api_key: 'b6195a0e1324007c9d596bc70d15d3a5',
    api_secret: 'cf8e1c69f540f109318d16aea2afea10',
    mapping: ['Customer.email', 'Company.name'],
    custom_properties: ['Campaign Source', 'plan', 'tutorial complete']
  }
}
```

<img width="1220" alt="screen shot 2018-07-05 at 15 42 54" src="https://user-images.githubusercontent.com/664261/42326726-460171ae-806a-11e8-9127-95fb6af9df76.png">